docs/posts/2025/08/interactive-shell-demo-bzip.cast

```diff
@@ -0,0 +1,141 @@
+[20.765161, "o", "\r\u001b[2K\u001b[32m●∙∙\u001b[0m \u001b[1;32mInstalling bzip2/1.0.8: building (6 out of 17 steps done)\u001b[0m \u001b[38;2;249;38;114m━━━━━━━━━━━━━\u001b[0m\u001b[38;5;237m╺\u001b[0m\u001b[38;5;237m━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[33m0:00:01\u001b[0m"]
+[21.017541, "o", "\r\u001b[2K\u001b[32m∙●∙\u001b[0m \u001b[1;32mInstalling bzip2/1.0.8: building (6 out of 17 steps done)\u001b[0m \u001b[38;2;249;38;114m━━━━━━━━━━━━━\u001b[0m\u001b[38;5;237m╺\u001b[0m\u001b[38;5;237m━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[33m0:00:01\u001b[0m"]
+[21.269535, "o", "\r\u001b[2K\u001b[32m∙●∙\u001b[0m \u001b[1;32mInstalling bzip2/1.0.8: building (6 out of 17 steps done)\u001b[0m \u001b[38;2;249;38;114m━━━━━━━━━━━━━\u001b[0m\u001b[38;5;237m╺\u001b[0m\u001b[38;5;237m━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[33m0:00:01\u001b[0m"]
+[21.52077, "o", "\r\u001b[2K\u001b[32m∙●∙\u001b[0m \u001b[1;32mInstalling bzip2/1.0.8: building (6 out of 17 steps done)\u001b[0m \u001b[38;2;249;38;114m━━━━━━━━━━━━━\u001b[0m\u001b[38;5;237m╺\u001b[0m\u001b[38;5;237m━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[33m0:00:01\u001b[0m"]
+[21.592466, "o", "\r\n\r\n\u001b[1;31mERROR: Shell command failed!\u001b[0m\r\n    full command              ->  make  -j 12 CC=gcc CFLAGS='-Wall -Winline -O3 -fPIC -g $(BIGFILES)'\r\n    exit code                 ->  2\r\n    called from               ->  'build_step' function in \r\n/home/eb/easybuild-easyblocks/easybuild/easyblocks/generic/configuremake.py (line 382)\r\n    working directory         ->  /dev/shm/eb/bzip2/1.0.8/system-system/bzip2-1.0.8\r\n\u001b[1;33m    output (stdout + stderr)  ->  /tmp/eb-_u1xuem_/run-shell-cmd-output/make-z6u6u46b/out.txt\u001b[0m\r\n\u001b[1;33m    interactive shell script  ->  /tmp/eb-_u1xuem_/run-shell-cmd-output/make-z6u6u46b/cmd.sh\u001b[0m\r\n\r\n"]
+[21.595034, "o", "\r\u001b[2K== ... (took 1 secs)\r\n\u001b[32m∙●∙\u001b[0m \u001b[1;32mInstalling bzip2/1.0.8: building (6 out of 17 steps done)\u001b[0m \u001b[38;2;249;38;114m━━━━━━━━━━━━━\u001b[0m\u001b[38;5;237m╺\u001b[0m\u001b[38;5;237m━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[33m0:00:01\u001b[0m"]
+[21.596114, "o", "\r\u001b[2K== FAILED: Installation ended unsuccessfully: shell command 'make ...' failed with exit code 2 in build step\r\nfor bzip2-1.0.8.eb (took 1 secs)\r\n"]
+[21.596739, "o", "\r\u001b[2K== Results of the build can be found in the log file(s) \r\n/tmp/eb-_u1xuem_/easybuild-bzip2-1.0.8-20250809.180309.EyMrP.log\r\n"]
+[21.601389, "o", "\r\u001b[2K== Summary:\r\n   * [FAILED]  bzip2/1.0.8\r\n"]
+[21.603399, "o", "\r\u001b[2K\r\n\u001b[?25hERROR: Installation of bzip2-1.0.8.eb failed: \"shell command 'make ...' failed with exit code 2 in build step for bzip2-1.0.8.eb\"\r\n"]
+[21.65737, "o", "\u001b[?2004h$ "]
+[35.335519, "o", "\u0007"]
```

Comment:
Why is there a 14sec gap here where nothing happens?

Reply:
6a6289d makes the pauses more even.